.gitignore

```
@@ -0,0 +1,35 @@
+
+.DS_Store
+CA-middle-daemon-storage/.DS_Store
+CA-middle-daemon-storage/Daily Keys/.DS_Store
+CA-middle-daemon-storage/Daily Keys/foo/daily_key.txt
+CA-middle-daemon-storage/Encrypted Certificates/.DS_Store
+CA-middle-daemon-storage/Encrypted Certificates/foo/enc_cert0.txt
+CA-middle-daemon-storage/Encrypted Certificates/foo/enc_cert1.txt
+CA-middle-daemon-storage/Encrypted Certificates/foo/enc_cert2.txt
+CA-middle-daemon-storage/Hashlists/.DS_Store
+CA-middle-daemon-storage/Hashlists/foo/hashlist.txt
+CA/.DS_Store
+CA/script/CA
+CA/storage/.DS_Store
+CA/storage/Domain Certificates/.DS_Store
+CA/storage/Domain Certificates/foo/cert0.pem
+CA/storage/Domain Certificates/foo/cert1.pem
+CA/storage/Domain Certificates/foo/cert2.pem
+CA/storage/Merkle Roots/.DS_Store
+CA/storage/Merkle Roots/foo/merkleroot.txt
+CA/storage/Precertificate/.DS_Store
+CA/storage/Precertificate/foo/cert.pem
+middle-daemon-website-daemon-storage/.DS_Store
+middle-daemon-website-daemon-storage/Daily Cert Verification/merkleroot.txt
+middle-daemon-website-daemon-storage/Daily Cert Verification/proof.pickle
+middle-daemon-website-daemon-storage/Daily Certificates/.DS_Store
+middle-daemon-website-daemon-storage/Daily Certificates/foo/certday1.pem
+middle-daemon/.DS_Store
+middle-daemon/script/go.sum
+middle-daemon/script/Middle_daemon
+website-daemon/.DS_Store
+website-daemon/script/Website-daemon
+website-daemon/storage/.DS_Store
+website-daemon/storage/Daily Cert Verification/cert_hash.txt
+website-daemon/storage/result/result.txt
```

```suggestion
*.DS_Store
*go.sum
*.pem
*.pickle
*.txt
```
Are you sure you wanna ignore 'CA/script/CA', `middle-daemon/script/Middle_daemon` and `website-daemon/script/Website-daemon`?

---

You mean the executable binary? We can always get it by running `go build`. They don't need to be included i think.

---

Can you not add the `.DS_Store` in this PR? You can do `git rm`, `git commit` and then `git push`

---

i think delete all of them. could you confirm?

---

Yeap thank you!!